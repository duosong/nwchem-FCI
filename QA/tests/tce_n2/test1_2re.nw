start tce_n2_1re
 
echo
 
geometry units bohr
  symmetry c1
  n 0 0   -2.068
  n 0 0    2.068
end
 
basis
 * library cc-pvdz
end

 
scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end
 
tce
ccsd
lshift 0.2
end
 
task tce energy
 
