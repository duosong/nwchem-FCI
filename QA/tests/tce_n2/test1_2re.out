nohup: ignoring input
 argument  1 = test1_2re.nw
  NWChem w/ OpenMP: maximum threads =    1



============================== echo of input deck ==============================
start tce_n2_1re

echo

geometry units bohr
  symmetry c1
  n 0 0   -2.068
  n 0 0    2.068
end

basis
 * library cc-pvdz
end


scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end

tce
ccsd
lshift 0.2
end

task tce energy

================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.0.1
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2020
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = koothan
    program         = /hpc/home/kowalski/nwchem/bin/LINUX64/nwchem
    date            = Wed Apr  7 16:10:56 2021

    compiled        = Sun_Jan_17_18:37:55_2021
    source          = /hpc/home/kowalski/nwchem
    nwchem branch   = 7.0.0
    nwchem revision = nwchem_on_git-2069-ga5c152f2d
    ga revision     = 5.8.0
    use scalapack   = T
    input           = test1_2re.nw
    prefix          = tce_n2_1re.
    data base       = ./tce_n2_1re.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107200 doubles =    100.0 Mbytes
    stack    =   13107197 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------
     1 autoz failed with cvr_scaling = 1.2 changing to 1.3
     2 autoz failed with cvr_scaling = 1.3 changing to 1.4
     3 autoz failed with cvr_scaling = 1.4 changing to 1.5
     4 autoz failed with cvr_scaling = 1.5 changing to 1.6
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 n                    7.0000     0.00000000     0.00000000    -2.06800000
    2 n                    7.0000     0.00000000     0.00000000     2.06800000

      Atomic Mass 
      ----------- 

      n                 14.003070


 Effective nuclear repulsion energy (a.u.)      11.8471953578

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       2.18868


            XYZ format geometry
            -------------------
     2
 geometry
 n                     0.00000000     0.00000000    -1.09433855
 n                     0.00000000     0.00000000     1.09433855



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvdz                   on all atoms 


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  n (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  9.04600000E+03  0.000700
  1 S  1.35700000E+03  0.005389
  1 S  3.09300000E+02  0.027406
  1 S  8.77300000E+01  0.103207
  1 S  2.85600000E+01  0.278723
  1 S  1.02100000E+01  0.448540
  1 S  3.83800000E+00  0.278238
  1 S  7.46600000E-01  0.015440

  2 S  9.04600000E+03 -0.000153
  2 S  1.35700000E+03 -0.001208
  2 S  3.09300000E+02 -0.005992
  2 S  8.77300000E+01 -0.024544
  2 S  2.85600000E+01 -0.067459
  2 S  1.02100000E+01 -0.158078
  2 S  3.83800000E+00 -0.121831
  2 S  7.46600000E-01  0.549003

  3 S  2.24800000E-01  1.000000

  4 P  1.35500000E+01  0.039919
  4 P  2.91700000E+00  0.217169
  4 P  7.97300000E-01  0.510319

  5 P  2.18500000E-01  1.000000

  6 D  8.17000000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 n                          cc-pvdz                  6       15   3s2p1d


                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    30
  atoms           =     2
  closed shells   =     7
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./tce_n2_1re.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 n                          cc-pvdz                  6       15   3s2p1d



 Forming initial guess at       0.2s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -108.60004629

      Non-variational initial energy
      ------------------------------

 Total energy =    -107.682259
 1-e energy   =    -171.391947
 2-e energy   =      51.862493
 HOMO         =      -0.224504
 LUMO         =      -0.160445


 Starting SCF solution at       0.2s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-10
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-10
 ----------------------------------------------


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -108.2269145217  4.15D-01  1.41D-01      0.2
                 2     -108.2368944333  1.02D-01  4.31D-02      0.2
                 3     -108.2378487642  2.01D-03  1.03D-03      0.3
                 4     -108.2378490659  1.50D-06  9.54D-07      0.5
                 5     -108.2378490659  6.88D-12  5.00D-12      0.7


       Final RHF  results 
       ------------------ 

         Total SCF energy =   -108.237849065854
      One-electron energy =   -170.333512920867
      Two-electron energy =     50.248468497180
 Nuclear repulsion energy =     11.847195357834

        Time for solution =      0.6s


             Final eigenvalues
             -----------------

              1      
    1  -15.8273
    2  -15.8271
    3   -1.0730
    4   -0.9815
    5   -0.4465
    6   -0.3815
    7   -0.3815
    8   -0.1042
    9   -0.1042
   10   -0.0093
   11    0.8342
   12    0.8707
   13    0.8707
   14    0.8718
   15    0.9858
   16    0.9858
   17    1.0171

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    2  Occ=2.000000D+00  E=-1.582709D+01
              MO Center=  1.5D-19, -7.7D-21,  4.6D-10, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.707406  1 N  s                 16     -0.707406  2 N  s          

 Vector    3  Occ=2.000000D+00  E=-1.072977D+00
              MO Center= -6.1D-17,  8.2D-17, -6.8D-13, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.376513  1 N  s                 18      0.376513  2 N  s          
     2      0.363117  1 N  s                 17      0.363117  2 N  s          

 Vector    4  Occ=2.000000D+00  E=-9.815089D-01
              MO Center= -5.0D-17,  7.0D-18,  6.5D-13, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.396181  1 N  s                 18     -0.396181  2 N  s          
     2      0.389149  1 N  s                 17     -0.389149  2 N  s          

 Vector    5  Occ=2.000000D+00  E=-4.464925D-01
              MO Center=  2.8D-17,  9.6D-17,  4.1D-14, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.414681  1 N  pz                21     -0.414681  2 N  pz         
     9      0.327644  1 N  pz                24     -0.327644  2 N  pz         

 Vector    6  Occ=2.000000D+00  E=-3.814541D-01
              MO Center=  4.2D-16, -1.3D-16,  6.0D-14, r^2= 1.9D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.416524  1 N  px                19      0.416524  2 N  px         
     7      0.365612  1 N  px                22      0.365612  2 N  px         

 Vector    7  Occ=2.000000D+00  E=-3.814541D-01
              MO Center=  4.0D-17, -1.4D-16,  4.7D-14, r^2= 1.9D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.416524  1 N  py                20      0.416524  2 N  py         
     8      0.365612  1 N  py                23      0.365612  2 N  py         

 Vector    8  Occ=0.000000D+00  E=-1.042171D-01
              MO Center= -1.0D-16,  1.4D-17, -6.1D-14, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.435776  1 N  px                19     -0.435776  2 N  px         
     7      0.382671  1 N  px                22     -0.382671  2 N  px         

 Vector    9  Occ=0.000000D+00  E=-1.042171D-01
              MO Center= -1.8D-17, -4.9D-17, -4.9D-14, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.435776  1 N  py                20     -0.435776  2 N  py         
     8      0.382671  1 N  py                23     -0.382671  2 N  py         

 Vector   10  Occ=0.000000D+00  E=-9.314212D-03
              MO Center=  1.4D-16, -7.9D-18, -1.3D-14, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.471822  1 N  pz                21      0.471822  2 N  pz         
     9      0.457811  1 N  pz                24      0.457811  2 N  pz         

 Vector   11  Occ=0.000000D+00  E= 8.342132D-01
              MO Center=  1.4D-14,  1.4D-14,  3.8D-14, r^2= 3.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.248717  1 N  s                 18      1.248717  2 N  s          
     2     -0.877071  1 N  s                 17     -0.877071  2 N  s          
     1     -0.396467  1 N  s                 16     -0.396467  2 N  s          
    10     -0.196315  1 N  dxx               13     -0.196315  1 N  dyy        
    25     -0.196315  2 N  dxx               28     -0.196315  2 N  dyy        

 Vector   12  Occ=0.000000D+00  E= 8.706658D-01
              MO Center= -1.5D-15, -1.6D-14, -4.8D-13, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.703832  1 N  py                20      0.703832  2 N  py         
     8     -0.666245  1 N  py                23     -0.666245  2 N  py         

 Vector   13  Occ=0.000000D+00  E= 8.706658D-01
              MO Center= -1.3D-14,  1.2D-15, -5.3D-13, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.703832  1 N  px                19      0.703832  2 N  px         
     7     -0.666245  1 N  px                22     -0.666245  2 N  px         

 Vector   14  Occ=0.000000D+00  E= 8.717525D-01
              MO Center=  1.0D-15,  1.8D-16, -3.2D-13, r^2= 4.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.813255  1 N  s                 18     -0.813255  2 N  s          
     9     -0.614370  1 N  pz                24     -0.614370  2 N  pz         
     2     -0.592871  1 N  s                 17      0.592871  2 N  s          
     6      0.519575  1 N  pz                21      0.519575  2 N  pz         
     1     -0.268697  1 N  s                 16      0.268697  2 N  s          

 Vector   15  Occ=0.000000D+00  E= 9.858038D-01
              MO Center=  1.5D-17, -7.9D-18,  4.8D-13, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.803954  1 N  py                23     -0.803954  2 N  py         
     5     -0.695967  1 N  py                20      0.695967  2 N  py         
     7      0.162269  1 N  px                22     -0.162269  2 N  px         

 Vector   16  Occ=0.000000D+00  E= 9.858038D-01
              MO Center= -5.2D-16,  1.5D-16,  5.2D-13, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.803954  1 N  px                22     -0.803954  2 N  px         
     4     -0.695967  1 N  px                19      0.695967  2 N  px         
     8     -0.162269  1 N  py                23      0.162269  2 N  py         

 Vector   17  Occ=0.000000D+00  E= 1.017054D+00
              MO Center= -4.9D-16,  4.1D-16,  2.6D-13, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.730456  1 N  pz                21     -0.730456  2 N  pz         
     9     -0.630811  1 N  pz                24      0.630811  2 N  pz         


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
         119.771730471360           0.000000000000           0.000000000000
           0.000000000000         119.771730471360           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 N    7     7.00   2.00  0.98  1.01  1.57  1.45 -0.01
    2 N    7     7.00   2.00  0.98  1.01  1.57  1.45 -0.01

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0     -0.000000      0.000000     14.000000

     1   1 0 0     -0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1     -0.000000      0.000000      0.000000

     2   2 0 0     -8.709176      0.000000      0.000000
     2   1 1 0     -0.000000      0.000000      0.000000
     2   1 0 1     -0.000000      0.000000      0.000000
     2   0 2 0     -8.709176      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -7.892825      0.000000     59.872736

                   NWChem Extensible Many-Electron Theory Module
                   ---------------------------------------------

              ======================================================
                   This portion of the program was automatically
                  generated by a Tensor Contraction Engine (TCE).
                  The development of this portion of the program
                 and TCE was supported by US Department of Energy,
                Office of Science, Office of Basic Energy Science.
                      TCE is a product of Battelle and PNNL.
              Please cite: S.Hirata, J.Phys.Chem.A 107, 9887 (2003).
              ======================================================

            General Information
            -------------------
      Number of processors :     1
         Wavefunction type : Restricted Hartree-Fock
          No. of electrons :    14
           Alpha electrons :     7
            Beta electrons :     7
           No. of orbitals :    60
            Alpha orbitals :    30
             Beta orbitals :    30
        Alpha frozen cores :     0
         Beta frozen cores :     0
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity : singlet 
    Number of AO functions :    30
       Number of AO shells :    12
        Use of symmetry is : off
      Symmetry adaption is : off
         Schwarz screening : 0.10D-09

          Correlation Information
          -----------------------
          Calculation type : Coupled-cluster singles & doubles                           
   Perturbative correction : none                                                        
            Max iterations :      100
        Residual threshold : 0.10D-06
     T(0) DIIS level shift : 0.20D+00
     L(0) DIIS level shift : 0.00D+00
     T(1) DIIS level shift : 0.00D+00
     L(1) DIIS level shift : 0.00D+00
     T(R) DIIS level shift : 0.00D+00
     T(I) DIIS level shift : 0.00D+00
   CC-T/L Amplitude update :  5-th order DIIS
                I/O scheme : Global Array Library
        L-threshold :  0.10D-06
        EOM-threshold :  0.10D-06
 no EOMCCSD initial starts read in
 TCE RESTART OPTIONS
 READ_INT:   F
 WRITE_INT:  F
 READ_TA:    F
 WRITE_TA:   F
 READ_XA:    F
 WRITE_XA:   F
 READ_IN3:   F
 WRITE_IN3:  F
 SLICE:      F
 D4D5:       F
 quantumc:   F
 qrotation:  F
 pwtce:      F
 qducc:      F
 nacto:                         0
 nrotv:                         0
 nrot:                          0

            Memory Information
            ------------------
          Available GA space size is      26213500 doubles
          Available MA space size is      26212676 doubles

 Maximum block size        32 doubles

 tile_dim =     23

 Block   Spin    Irrep     Size     Offset   Alpha
 -------------------------------------------------
   1    alpha     a      7 doubles       0       1
   2    beta      a      7 doubles       7       1
   3    alpha     a     23 doubles      14       3
   4    beta      a     23 doubles      37       3

 Global array virtual files algorithm will be used

 Parallel file system coherency ......... OK

 Fock matrix recomputed
 1-e file size   =              900
 1-e file name   = ./tce_n2_1re.f1
 Cpu & wall time / sec            0.0            0.0

 tce_ao2e: fast2e=1
 half-transformed integrals in memory

 2-e (intermediate) file size =         2140200
 2-e (intermediate) file name = ./tce_n2_1re.v2i
 Cpu & wall time / sec            0.2            0.2

 tce_mo2e: fast2e=1
 2-e integrals stored in memory

 2-e file size   =          1356121
 2-e file name   = ./tce_n2_1re.v2
 Cpu & wall time / sec            0.0            0.0
 T1-number-of-tasks                    1

 t1 file size   =              161
 t1 file name   = ./tce_n2_1re.t1
 t1 file handle =       -999
 T2-number-of-boxes                    2

 t2 file size   =            51842
 t2 file name   = ./tce_n2_1re.t2
 t2 file handle =       -996
CCSD iterations
 ---------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall 
 ---------------------------------------------------------
NEW TASK SCHEDULING
CCSD_T1_NTS --- OK
CCSD_T2_NTS --- OK
    1   1.0902832913249  -1.0133526472487     0.0     0.0
    2   0.8471433863048  -0.2180453918531     0.0     0.0
    3   0.6260273135097  -0.8236133459514     0.0     0.0
    4   0.4771236580640  -0.3779179729099     0.0     0.0
    5   0.3381844281360  -0.7115096593232     0.0     0.0
 MICROCYCLE DIIS UPDATE:                    5                    5
    6   0.0657877861070  -0.5779636719513     0.0     0.0
    7   0.0464649557463  -0.5979102545802     0.0     0.0
    8   0.0504830586325  -0.5949660422687     0.0     0.0
    9   0.0408459177645  -0.6086876652852     0.0     0.0
   10   0.0424912429095  -0.6100098726518     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   10                    5
   11   0.0502045380619  -0.6930925242449     0.0     0.0
   12   0.0547678591347  -0.7254440698689     0.0     0.0
   13   0.0268775584744  -0.6964751608487     0.0     0.0
   14   0.0244733496837  -0.7102353921604     0.0     0.0
   15   0.0116140007946  -0.6981407251676     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   15                    5
   16   0.0065022047964  -0.6976979202705     0.0     0.0
   17   0.0065248458342  -0.6976922374747     0.0     0.0
   18   0.0044381407902  -0.6956177043116     0.0     0.0
   19   0.0044618521110  -0.6953264698162     0.0     0.0
   20   0.0034005046620  -0.6941250687749     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   20                    5
   21   0.0010101036286  -0.6895643678151     0.0     0.0
   22   0.0008070945028  -0.6896089008710     0.0     0.0
   23   0.0005788917103  -0.6894507903131     0.0     0.0
   24   0.0005072079212  -0.6894364753309     0.0     0.0
   25   0.0003883195389  -0.6893394878848     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   25                    5
   26   0.0001319776647  -0.6890321662512     0.0     0.0
   27   0.0001064036621  -0.6890243259153     0.0     0.0
   28   0.0000816329827  -0.6889971690420     0.0     0.0
   29   0.0000765840528  -0.6889875128883     0.0     0.0
   30   0.0000622585691  -0.6889697616905     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   30                    5
   31   0.0000228433964  -0.6888915129880     0.0     0.0
   32   0.0000178067870  -0.6888913563713     0.0     0.0
   33   0.0000132998903  -0.6888878878361     0.0     0.0
   34   0.0000116382943  -0.6888870133850     0.0     0.0
   35   0.0000091420601  -0.6888848304873     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   35                    5
   36   0.0000037933469  -0.6888781254298     0.0     0.0
   37   0.0000032437168  -0.6888778884052     0.0     0.0
   38   0.0000025211105  -0.6888770895582     0.0     0.0
   39   0.0000023206909  -0.6888767786333     0.0     0.0
   40   0.0000018943187  -0.6888762556597     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   40                    5
   41   0.0000006809640  -0.6888739361436     0.0     0.0
   42   0.0000005456301  -0.6888739484302     0.0     0.0
   43   0.0000003980330  -0.6888738235405     0.0     0.0
   44   0.0000003618338  -0.6888738017088     0.0     0.0
   45   0.0000002805750  -0.6888737249967     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   45                    5
   46   0.0000001291603  -0.6888735064090     0.0     0.0
   47   0.0000001095795  -0.6888734976093     0.0     0.0
   48   0.0000000849679  -0.6888734715599     0.0     0.0
 -----------------------------------------------------------------
 Iterations converged
 CCSD correlation energy / hartree =        -0.688873471559910
 CCSD total energy / hartree       =      -108.926722537414193

 Singles contributions

 Doubles contributions
     8a   (alpha)     9a   (alpha) ---     6a   (alpha)     7a   (alpha)        0.1188305975
     8a   (alpha)     9a   (alpha) ---     7a   (alpha)     6a   (alpha)       -0.1188305975
     9a   (alpha)     8a   (alpha) ---     6a   (alpha)     7a   (alpha)       -0.1188305975
     9a   (alpha)     8a   (alpha) ---     7a   (alpha)     6a   (alpha)        0.1188305975
     8a   (alpha)     8a   (beta ) ---     6a   (alpha)     6a   (beta )       -0.6096364757
     8a   (alpha)     9a   (beta ) ---     6a   (alpha)     7a   (beta )       -0.2172700400
     8a   (alpha)     9a   (beta ) ---     7a   (alpha)     6a   (beta )       -0.3361006375
     8a   (alpha)    10a   (beta ) ---     5a   (alpha)     6a   (beta )       -0.2356470060
     8a   (alpha)    10a   (beta ) ---     6a   (alpha)     5a   (beta )       -0.1591750300
     9a   (alpha)     8a   (beta ) ---     6a   (alpha)     7a   (beta )       -0.3361006375
     9a   (alpha)     8a   (beta ) ---     7a   (alpha)     6a   (beta )       -0.2172700400
     9a   (alpha)     9a   (beta ) ---     7a   (alpha)     7a   (beta )       -0.6096364757
     9a   (alpha)    10a   (beta ) ---     5a   (alpha)     7a   (beta )       -0.2356470060
     9a   (alpha)    10a   (beta ) ---     7a   (alpha)     5a   (beta )       -0.1591750300
    10a   (alpha)     8a   (beta ) ---     5a   (alpha)     6a   (beta )       -0.1591750300
    10a   (alpha)     8a   (beta ) ---     6a   (alpha)     5a   (beta )       -0.2356470060
    10a   (alpha)     9a   (beta ) ---     5a   (alpha)     7a   (beta )       -0.1591750300
    10a   (alpha)     9a   (beta ) ---     7a   (alpha)     5a   (beta )       -0.2356470060
    10a   (alpha)    10a   (beta ) ---     5a   (alpha)     5a   (beta )       -0.3602911372

 Task  times  cpu:        1.8s     wall:        2.0s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls: 2594     2594     2.35e+04 3881     8559        0        0     1.29e+04 
number of processes/call 3.66e+14 -6.07e+14 4.44e+14 0.00e+00 0.00e+00
bytes total:             2.41e+09 1.13e+08 6.66e+08 0.00e+00 0.00e+00 1.04e+05
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 27984976 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        16	        36
	current total bytes		         0	         0
	maximum total bytes		     80104	  22509768
	maximum total K-bytes		        81	     22510
	maximum total M-bytes		         1	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

          E. Aprà, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, Y. Alexeev, J. Anchell,
       V. Anisimov, F. W. Aquino, R. Atta-Fynn, J. Autschbach, N. P. Bauman,
     J. C. Becca, D. E. Bernholdt, K. Bhaskaran-Nair, S. Bogatko, P. Borowski,
        J. Boschen, J. Brabec, A. Bruner, E. Cauët, Y. Chen, G. N. Chuev,
      C. J. Cramer, J. Daily, M. J. O. Deegan, T. H. Dunning Jr., M. Dupuis,
   K. G. Dyall, G. I. Fann, S. A. Fischer, A. Fonari, H. Früchtl, L. Gagliardi,
      J. Garza, N. Gawande, S. Ghosh, K. Glaesemann, A. W. Götz, J. Hammond,
       V. Helms, E. D. Hermes, K. Hirao, S. Hirata, M. Jacquelin, L. Jensen,
   B. G. Johnson, H. Jónsson, R. A. Kendall, M. Klemm, R. Kobayashi, V. Konkov,
      S. Krishnamoorthy, M. Krishnan, Z. Lin, R. D. Lins, R. J. Littlefield,
      A. J. Logsdail, K. Lopata, W. Ma, A. V. Marenich, J. Martin del Campo,
   D. Mejia-Rodriguez, J. E. Moore, J. M. Mullin, T. Nakajima, D. R. Nascimento,
    J. A. Nichols, P. J. Nichols, J. Nieplocha, A. Otero-de-la-Roza, B. Palmer,
    A. Panyala, T. Pirojsirikul, B. Peng, R. Peverati, J. Pittner, L. Pollack,
   R. M. Richard, P. Sadayappan, G. C. Schatz, W. A. Shelton, D. W. Silverstein,
   D. M. A. Smith, T. A. Soares, D. Song, M. Swart, H. L. Taylor, G. S. Thomas,
            V. Tipparaju, D. G. Truhlar, K. Tsemekhman, T. Van Voorhis,
     Á. Vázquez-Mayagoitia, P. Verma, O. Villa, A. Vishnu, K. D. Vogiatzis,
        D. Wang, J. H. Weare, M. J. Williamson, T. L. Windus, K. Woliński,
        A. T. Wong, Q. Wu, C. Yang, Q. Yu, M. Zacharias, Z. Zhang, Y. Zhao,
                                and R. J. Harrison
                        "NWChem: Past, present, and future
                         J. Chem. Phys. 152, 184102 (2020)
                               doi:10.1063/5.0004997

                                      AUTHORS
                                      -------
     E. Apra, E. J. Bylaska, N. Govind, K. Kowalski, M. Valiev, W. A. de Jong,
      T. P. Straatsma, H. J. J. van Dam, D. Wang, T. L. Windus, N. P. Bauman,
       A. Panyala, J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec,
    K. Lopata, S. A. Fischer, S. Krishnamoorthy, M. Jacquelin, W. Ma, M. Klemm,
       O. Villa, Y. Chen, V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler,
           Eric Hermes, L. Jensen, J. E. Moore, J. C. Becca, V. Konjkov,
            D. Mejia-Rodriguez, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
        P.-D. Fan, A. Fonari, M. J. Williamson, R. J. Harrison, J. R. Rehr,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
      M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
     G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols,
       K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski,
       T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood,
      E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju,
        R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        1.8s     wall:        2.1s
